assets/src/models/createDetourMachine.ts

```diff
@@ -40,6 +52,7 @@ export const createDetourMachine = setup({
       activatedAt?: Date
 
       editedSelectedDuration?: string
+      saved: boolean
```

Comment:
issue: making this required without adding this to the `serde` module I think would cause issues?

```suggestion
      saved?: boolean
```